CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -1265,7 +1277,7 @@ CFDictionaryRef __CFGetEnvironment() {
         extern char **environ;
         char **envp = environ;
 #elif DEPLOYMENT_TARGET_LINUX
-#ifndef environ
+#if !defined(environ) && !defined(__ANDROID__)
```

Comment:
I'm confused, if environ is already defined in Android, then the second half of your #if is never evaluated; if environ is undefined in Android, then what is envp on Android?

Am I missing something?